main/src/ca/uwaterloo/flix/language/phase/Weeder.scala

```diff
@@ -580,14 +587,11 @@ object Weeder {
   }
 
   /**
-    * Checks that the name's casing matches the alias's casing. (I.e., both uppercase or both lowercase.)
+    * Returns `true` if `ident` and `alias` are both lower/upper case.
```

Comment:
IMO you made this comment both concise/unclear

Reply:
Fixed.